src/adapter/src/session/vars.rs

```diff
@@ -360,6 +360,14 @@ static EMIT_TIMESTAMP_NOTICE: ServerVar<bool> = ServerVar {
     internal: false,
 };
 
+static EMIT_TRACING_NOTICE: ServerVar<bool> = ServerVar {
+    name: UncasedStr::new("emit_tracing_notice"),
```

Comment:
```suggestion
    name: UncasedStr::new("emit_trace_id_notice"),
```

Light preference to specify that this is about the trace id, not tracing in general.